src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -214,8 +215,12 @@ public CaseDetails submitCase(String authorization, CaseRequest caseRequest)
                 notificationsProperties.getSubmitCaseEmailTemplateId(),
                 caseData.getClaimantType().getClaimantEmailAddress(),
                 caseRequest.getCaseId(),
-                caseData.getClaimantIndType().getClaimantFirstNames(),
-                caseData.getClaimantIndType().getClaimantLastName(),
+                Strings.isNullOrEmpty(caseData.getClaimantIndType().getClaimantFirstNames())
+                    ? userInfo.getGivenName()
+                    : caseData.getClaimantIndType().getClaimantFirstNames(),
```

Comment:
This may still result in a null pointer. The first name and last name on case data comes from userInfo on case creation. If it doesn't exist on userinfo at creation I'm not sure it will exist on submission